### PR TITLE
Wines and grape juice

### PR DESCRIPTION
**Pull request by Gringonius**
-You can now juice both grapes and green grapes.
-Enzyme reaction with these fruit create wine and white wine, respectively.
-Berries enzyme reaction now creates berry wine.
Update: -Plump helmet wine and juice are now a thing.
[tested]

:cl:
 * rscadd: Added grape juices, white wine, berry wine and plump helmet wine, go get yourself drunk!

### CODE REVIEW

**Comment by SGuivenne** [comment]
Can we get one of these with Plump Helmets to make Plump Helmet Wine too?  That shits more dorfy than Ale.

**Comment by Gringonius** [comment]
Main reason I made this change is to please my OCD with wine being actually made from grapes and to enable a little more variety for future dishes with the white wine. I could add plump helmets but then I could use some ideas for dorf food that uses it for cooking. After all you can just use the goof still to make wine with a plump label. But I guess that berry wine may be a dead end too.

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -5189,6 +5203,24 @@
...
+	description = "Sweet berry wine!"
+	color = "#C760A2" //rgb: 199, 96, 162
+	dizzy_adj = 2
+	slur_start = 65
+	confused_start = 145
+
+/datum/reagent/ethanol/wwine
+	name = "White Wine"
+	id = WWINE
+	description = "An premium alchoholic beverage made from distilled green grape juice."
```
Comment (MadmanMartian) [inline code/modules/reagents/Chemistry-Reagents.dm]: A premium alcoholic beverage made from distilled green grape juice.

Comment (Gringonius) [reply code/modules/reagents/Chemistry-Reagents.dm]: Now that's an old typo nobody ever noticed.

**Comment by Gringonius** [comment]
Added plump helmet wine and juicing. Recipes for food and drinks with these is still going to be WIP.

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 2]
```diff
@@ -5183,16 +5204,43 @@
...
+	description = "Sweet berry wine!"
+	color = "#C760A2" //rgb: 199, 96, 162
+	dizzy_adj = 2
+	slur_start = 65
+	confused_start = 145
+
+/datum/reagent/ethanol/wwine
+	name = "White Wine"
+	id = WWINE
+	description = "An premium alcoholic beverage made from fermented green grape juice."
```
Comment (MadmanMartian) [inline code/modules/reagents/Chemistry-Reagents.dm]: A premium(..)

Comment (Gringonius) [reply code/modules/reagents/Chemistry-Reagents.dm]: Ree

**Review by MadmanMartian — APPROVED** [review]
personally I reckon that plump helm wine sprite could use some TLC (Froth, make it less flat) but this all works

**Pull request merged**